UICatalog/Scenarios/AutoScrollingChat.cs

```diff
@@ -0,0 +1,137 @@
+using System;
+using System.Collections.Generic;
+using System.Data;
+using Terminal.Gui;
+using System.Linq;
+using System.Globalization;
+using System.IO;
+using System.Text;
+using NStack;
+using System.Text.RegularExpressions;
+using Terminal.Gui.Graphs;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace UICatalog.Scenarios {
+
+	[ScenarioMetadata (Name: "Auto Scrolling Text", Description: "Demonstrates async update and auto scrolling.")]
+	[ScenarioCategory ("Controls")]
+	[ScenarioCategory ("Text and Formatting")]
+	public class AutoScrollingChat : Scenario 
+	{
+        const string Words = "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum";
+		const string Users = "Bot232 Frank Mary Harley";
+		private TextView textView;
+
+		CancellationTokenSource cts = new CancellationTokenSource();
+
+
+		public override void Setup ()
+		{
+			Win.Title = this.GetName();
+			textView = new TextView {
+				Width = Dim.Fill(),
+				Height = Dim.Fill(2),
+				ReadOnly = true
+			};
```

Comment:
The Scenario has a usability issue if mouse is used: There's no scroll bar. 

Similarly, there's no visual indication that I can scroll (because there's no scrollbarview).

Can you update the demo to include a `ScrollBarView`?

Related: #1838